%YAML:1.0

N_SCAN: 16
Horizon_SCAN: 1800